cms/templates/asset_index.html

```diff
@@ -53,7 +53,7 @@ <h3 class="sr">${_("Page Actions")}</h3>
         <article class="content-primary" role="main">
             <div class="wrapper-assets" />
             <div class="ui-loading">
-                <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading&hellip;")}</span></p>
+                <p tabindex="0" aria-label="Loading"><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading&hellip;")}</span></p>
```

Comment:
@mbilalmughal Since the text "Loading```&hellip;```" is already considered "accessible text" and is a functional equivalent for the icon, no aria-label is needed here, and no tabindex.   Since you are in these files though, there is a related [JIRA issue](https://openedx.atlassian.net/browse/TNL-672?jql=text%20~%20%22hellip%22) @andy-armstrong  open regarding the use of ```&hellip```;
I recommend removing the tabindex and aria-label as well as the ```&hellip;```
I am also recommending that an ```aria-hidden="true"``` be added to the icon font i.e.:
```<i class="icon-refresh" aria-hidden="true"></i>```  Icon fonts can sometimes be focused on (In Voiceover) with nothing read to the screen reader user, which can be very confusing (because they may think they are missing important content).  Since "Loading" says it all, the icon is not necessary to expose to those users.
This feedback applies to all of the following changes except the last one.